include/status-code/status_code.hpp

```diff
@@ -421,8 +421,8 @@ namespace traits
   //! Determines whether the mixin contained in `StatusCode` contains non-static member variables.
   template <class StatusCode> using has_stateful_mixin = detail::has_stateful_mixin<typename detail::remove_cvref<StatusCode>::type::value_type>;
 
-  //! Determines whether the status code `From` can be type erased into a status code based on `ToValueType`.
-  template <class From, class ToValueType> using is_type_erasable_to = detail::is_domain_type_erasable_to<typename detail::remove_cvref<From>::type::domain_type, ToValueType>;
+  //! Determines whether the status code `From` can be type erased into the status code `To`.
+  template <class To, class From> using is_type_erasable_to = detail::domain_value_type_erasure_is_safe<typename detail::remove_cvref<To>::type::domain_type, typename detail::remove_cvref<From>::type::domain_type>;
```

Comment:
I intentionally chose the type parameter order `From, To` instead of `To, From`, because I think it is more ergonomic to use in compound requirements (like `std::convertible_to`).

Reply:
`std::is_convertible` is a weird one because it has flipped argument order to anything else like it in `type_traits`, to my best knowledge. I would wonder if the committee would do differently if it stepped back in time. Too late now of course.

As to which is better for this specific instance, I suspect it's an east vs west const debate i.e. there is no one true answer. It's like whether Result ought to model Optional or Variant, one will have opinions, but are they incontrovertibly the best choice?

Reply:
Fair enough.